### PR TITLE
Run codecov on all engines

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by axic** [comment]
@chfast can you have a look why codecov is failing?

**Comment by chfast** [comment]
The compiler went OOM.

File: cmake/ProjectWAVM.cmake [hunk 1]
```diff
@@ -26,6 +26,10 @@ set(other_libraries ${platform_library} ${wasm_library} ${ir_library} ${logging_
 
 set(flags "-Wno-error -fvisibility=hidden")
 
+if(CMAKE_GENERATOR STREQUAL Ninja)
+    set(build_command BUILD_COMMAND ninja -j1)
```
Comment (axic) [inline cmake/ProjectWAVM.cmake]: Any way to use the concurrency flag from cmake (if there's one)?

**Comment by axic** [comment]
@chfast would it be a lot of work splitting up coverage into:
- running all engines (this will not allow us to see if the eei is covered properly)
- running only wabt to see if it covers the entire eei properly

File: cmake/ProjectWAVM.cmake (hunk 1 above)
Comment (chfast) [reply cmake/ProjectWAVM.cmake]: Somehow this flag is not passed down to external projects for ninja. We can switch to make to make is simple.

**Comment by chfast** [comment]
> @chfast would it be a lot of work splitting up coverage into:
> 
>     * running all engines (this will not allow us to see if the eei is covered properly)
> 
>     * running only wabt to see if it covers the entire eei properly

A bit. On codecov you can tag different data uploads, but codecov is not able to filter these in web UI. Maybe coveralls can do it.

**Comment by axic** [comment]
@chfast thanks for fixing this!

**Pull request merged**